src/ooxml/java/org/apache/poi/xssf/streaming/SheetDataWriter.java

```diff
@@ -354,88 +355,54 @@ protected void outputQuotedString(String s) throws IOException {
             return;
         }
 
-        char[] chars = s.toCharArray();
-        int last = 0;
-        int length = s.length();
-        for (int counter = 0; counter < length; counter++) {
-            char c = chars[counter];
-            switch (c) {
-                case '<':
-                    writeLastChars(_out, chars, last, counter);
-                    last = counter + 1;
+        final String xa0 = Character.toString((char)0xa0);
```

Comment:
What is special about this character? Non-breaking space, right? Maybe a better short variable name would be nbsp for those of us familiar with HTML.

Reply:
Is Character.toString different than

    String nbsp = "\u00A0";

Reply:
this wasn't actually used - you have to use constants for matches in switch statements